lib/github_webhook/processor.rb

```diff
@@ -64,7 +64,7 @@ class UnsupportedContentTypeError < StandardError; end
   )
 
   def create
-    if self.respond_to? event_method
+    if self.respond_to?(event_method, true)
```

Comment:
So basically you're saying that'd you rather have all the `github_` methods private, do I understand that correctly? Because this line is not related to `:webhook_secret`.

Reply:
You're right, this affects the event method instead, but since it would to have it be private as well, I figured it would make sense to change this to work with private methods as well.